test/integration-tests/UniswapMixed.test.ts

```diff
@@ -279,6 +279,41 @@ describe('Uniswap V2, V3, and V4 Tests:', () => {
       expect(wethBalanceAfter.sub(wethBalanceBefore)).to.be.gte(minAmountOut1.add(minAmountOut2))
     })
 
+    it('PERMIT2 batch can silently fail', async () => {
+      const v2AmountIn1: BigNumber = expandTo18DecimalsBN(20)
+      const v2AmountIn2: BigNumber = expandTo18DecimalsBN(5)
+
+      const BATCH_PERMIT = {
+        details: [
+          {
+            token: DAI.address,
+            amount: v2AmountIn1,
+            expiration: 0, // expiration of 0 is block.timestamp
+            nonce: 0, // this is his first trade
+          },
+          {
+            token: WETH.address,
+            amount: v2AmountIn2,
+            expiration: 0, // expiration of 0 is block.timestamp
+            nonce: 0, // this is his first trade
+          },
+        ],
+        spender: router.address,
+        sigDeadline: DEADLINE,
+      }
+
+      const sig = await getPermitBatchSignature(BATCH_PERMIT, bob, permit2)
+
+      // transfer funds into DAI-USDC and DAI-USDT pairs to trade
+      // do not allow revert
+      planner.addCommand(CommandType.PERMIT2_PERMIT_BATCH, [BATCH_PERMIT, sig])
+
+      // allow revert
+      planner.addCommand(CommandType.PERMIT2_PERMIT_BATCH, [BATCH_PERMIT, sig], true)
+
+      await executeRouter(planner, bob, router, wethContract, daiContract, usdcContract)
```

Comment:
would love a check for what the new nonce is or something to show the tx succeeded.